File: android/src/main/kotlin/eu/kevin/kevin_flutter/entity/AccountSessionConfigurationEntity.kt
package eu.kevin.kevin_flutter.entity

import kotlinx.serialization.Serializable

@Serializable
internal data class AccountSessionConfigurationEntity(
    val state: String,
    val preselectedCountry: String?,
    val disableCountrySelection: Boolean,
    val countryFilter: List<String>,
    val bankFilter: List<String>,
    val preselectedBank: String?,
    val skipBankSelection: Boolean,
    val accountLinkingType: String,
)

File: packages/kevin_flutter_accounts/kevin_flutter_accounts_ios/ios/Classes/Entity/AccountSessionConfigurationEntity.swift
import Foundation

internal struct AccountSessionConfigurationEntity: Codable {
    let state: String
    let preselectedCountry: String?
    let disableCountrySelection: Bool
    let countryFilter: [String]
    let bankFilter: [String]
    let preselectedBank: String?
    let skipBankSelection: Bool
}
